packages/jumpstarter-driver-probe-rs/jumpstarter_driver_probe_rs/client.py

```
@@ -0,0 +1,93 @@
+        """Download a file to the device"""
+        with OpendalAdapter(client=self, operator=operator, path=path) as handle:
+            return self.call("download", handle)
+
+    def download_file(self, filepath):
+        """Download a local file to the device"""
+        absolute = Path(filepath).resolve()
+        return self.download(operator=Operator("fs", root="/"), path=str(absolute))
+
+    def read(self, width: int, address: int, words: int):
+        """Read from memory
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)